.travis.yml

```diff
@@ -43,7 +43,7 @@ deploy:
         branch: v1.8.47
     provider: pages
     skip-cleanup: true
-    github-token: 
```

Comment:
Unrelated change, but seems that whitespace at  line-endings wasn't previously trimmed.